tests/Integration.Tests/UcasLink/UcasUrlBuilderIntegrationTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using GovUk.Education.SearchAndCompare.Api.Ucas;
+using NUnit.Framework;
+
+namespace GovUk.Education.SearchAndCompare.Api.Integration.Tests.UcasLink
+{
+    /// <summary>
+    /// Set of tests for interacting with the live UCAS website,
+    /// in increasing order of pedanticness.
+    /// </summary>
+    [TestFixture]
```

Comment:
-    [Category("Integration")]
-    [Category("Integration_Ucas")]
-    [Explicit]

Can you add that so we can don't needless run the unit tests